libraries/rush-lib/src/cli/actions/ChangeAction.ts

```diff
@@ -130,6 +131,12 @@ export class ChangeAction extends BaseRushAction {
         `(or erroring in ${BULK_LONG_NAME} mode).`
     });
 
+    this._commitChangesFlagParameter = this.defineFlagParameter({
+      parameterLongName: '--commit',
+      parameterShortName: '-c',
+      description: `Commits the change files to git.`
```

Comment:
nit:

```suggestion
      description: `If this flag is specified generated changefiles will be commited automatically.`
```

A bit more consistent language with other flags.